src/com/lftechnology/java/training/srijan/basics/FabonacciNumber.java

```diff
@@ -0,0 +1,17 @@
+package com.lftechnology.java.training.srijan.basics;
+
+import java.util.logging.Logger;
+
+public class FabonacciNumber {
+
+	private static final Logger LOGGER = Logger.getLogger(FabonacciNumber.class.getName());
+
+	public static void main(String[] args) {
+		Fabonacci fabonacci = new Fabonacci(0, 1);
```

Comment:
0,1 are first and second number of fabonacci series. I have used a constructor to set the first and second number of fabonacci series